javascript/nanoexpress-pro/config.yaml

```diff
@@ -0,0 +1,12 @@
+framework:
+  website: nanoexpress/pro
```

Comment:
what is the website for `nanoexpress-pro` ?

Reply:
Oh sorry, it's github repo